### DISCUSSION TITLE
thanhle.blog/minh-hoc-duoc-gi-khi-tap-choi-c

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#95] [Ideas]
# thanhle.blog/minh-hoc-duoc-gi-khi-tap-choi-c

https://thanhle.blog/blog/minh-hoc-duoc-gi-khi-tap-choi-c

**Comment by hai123vn** [comment]
hehe bữa anh kêu là cuối tuần nhưng không nó là đầu tháng mới có blog mới

**Reply by thanhlmm** [reply]
😂

**Comment by thanhlmm** [comment]
Mình mới add thêm một phần `Bạn đi trước đối thủ mấy bước?`

Hic kêu là học được mà quên bài không nghĩ ra thì chắc là cũng không học được gì nhiều như chém gió rồi
